nifi-registry-utils/src/main/java/org/apache/nifi/registry/util/FileUtils.java

```diff
@@ -389,4 +389,38 @@ public static void sleepQuietly(final long millis) {
             /* do nothing */
         }
     }
+
+
+    // The invalid character list is copied from this Stackoverflow page.
+    // https://stackoverflow.com/questions/1155107/is-there-a-cross-platform-java-method-to-remove-filename-special-chars
+    private final static int[] INVALID_CHARS = {34, 60, 62, 124, 0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15,
```

Comment:
Should we add a space to this list? 

Technically a space is valid, but currently a bucket name of `This / is a / test` becomes `This _ is a _ test` but I was originally thinking it would be `This_is_a_test` or I guess technically it would have multiple underscores like `This___is_a___test` ?